pyproject.toml

```diff
@@ -33,6 +33,7 @@ pyvisa-py = {version ="0.5.3", optional = true}
 qm-qua = {version ="==1.1.1", optional = true}
 qualang-tools = {version ="==0.14.0", optional = true}
 laboneq = {version ="==2.4.0", optional = true}
+qibosoq = { git = "https://github.com/qiboteam/qibosoq.git", branch = "zcu111-flux-single-q" }
```

Comment:
```suggestion
qibosoq = { git = "https://github.com/qiboteam/qibosoq.git", branch = "zcu111-flux-single-q", optional = true}
```

Reply:
I'm leaving this comment open for now. After merging qibosoq into main, the branch parameter should be updated